ma_set.go

```diff
@@ -0,0 +1,10 @@
+package movavg
+
+// MA defines Moving Average calculator:
+// https://en.wikipedia.org/wiki/Moving_average
+type MASet interface {
```

Comment:
Not critiquing - just a perfectionist naming woes...

I think, that just `Set` name would be sufficient - package name is `movavg`.
So it's quite obvious that it'll be operating on moving averages.

So, roughly, it's a wrapper for `MA`. And `ThreadSafe` is a wrapper (well, it's not actually a "wrapper" but a constructor, but still).
`ThreadSafe(ma)` looks nicer than `MAThreadSafe(ma)`.

So I think that just `Set` name would be nicer.

(bikeshedding, sorry :smile: )